packages/esm-patient-biometrics-app/src/biometrics/biometrics-chart.component.scss

```diff
@@ -41,8 +41,47 @@
   width: 7.063rem;
 }
 
-.biometricSignsRadioButton {
-  @extend .bodyShort01;
-  color: $ui-05;
-  padding: $spacing-03 0;
+.verticalTabs {
+  margin: 1rem 0;
+  scroll-behavior: smooth;
+}
+
+.verticalTabs > ul {
+  flex-direction: column !important;
+}
+
+.tab:hover {
+  border-bottom: none !important;
+}
+
+.tab > button:first {
+  text-indent: 5rem;
+}
+
+.tab>button,
```

Comment:
Can we prettify this? Usually there are spaces here (`.tab > button`), see below. Consistency would be appreciated.

Same applies for many of the other lines, see e.g., line 80.

Reply:
Updated